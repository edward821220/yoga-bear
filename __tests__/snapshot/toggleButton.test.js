import { create } from "react-test-renderer";
import ToggleButton from "../../src/components/toggleButton";

/* eslint-disable */
describe("Jest Snapshot testing suite", () => {
  test("Matches DOM Snapshot", () => {
    const tree = create(<ToggleButton />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
