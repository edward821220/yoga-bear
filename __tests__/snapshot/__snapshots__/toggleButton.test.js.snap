// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Jest Snapshot testing suite Matches DOM Snapshot 1`] = `
<div
  className="sc-bcXHqe zqhWb"
>
  <input
    className="sc-dkrFOg LQqKO"
    data-testid="check-box"
    id="checkbox"
    type="checkbox"
  />
  <label
    className="sc-gswNZR iyRvYP"
    htmlFor="checkbox"
  />
</div>
`;
